rustplus/api/base_rust_api.py

```diff
@@ -142,6 +142,56 @@ async def send_wakeup_request(self) -> None:
 
         await self.remote.send_message(app_request)
 
+	async def switch_server(
+		self,
+		ip: str = None,
+		port: str = None,
+		steam_id: int = None,
+		player_token: int = None,
+		command_options: CommandOptions = None,
+		raise_ratelimit_exception: bool = True,
+		ratelimit_limit: int = 25,
+		ratelimit_refill: int = 3,
+		heartbeat: HeartBeat = None,
+		use_proxy: bool = False,
```

Comment:
It seems wasteful to have these "magic" values hard coded here should they change, maybe add some static config constants to the class? e.g. `CONFIG_RATELIMIT_LIMIT = 25`. It's only two values but does irk me a bit